protocol/simplified-guardian.md

```diff
@@ -0,0 +1,118 @@
+would require participation from a majority of the Security Council, we must either (1) allow a
+minority of the Security Council to unpause the system or (2) have the pause automatically expire
+after some period of time. Given the potential security concerns with allowing a minority of the
+Security Council to unpause, we prefer option (2).
+
+We therefore propose that the pause would expire after 6 months. Once the pause is used, the pause
+cannot be called again unless it is reset by a majority of the Security Council (this applies in
+all cases, regardless of whether the system unpauses due to expiry or due to explicit unpause).
+
+### Cluster-wide Pause
+
+Because the pause mechanism would become the main mechanism available to the Optimism Foundation,
+the pause mechanism would also become the basis for an updated incident response process. However,
+the Superchain-wide Pause is far too impactful to be the only available option during an incident.
+Given that we suspect many incidents will take the form of *single* impacted chains or chain
+clusters, it may be valuable to implement a Cluster-wide Pause on top of the Superchain-wide Pause.
+
+A chain would be paused if *either* the Superchain-wide Pause is active or its Cluster-wide Pause
+is active. We would implement the Cluster-wide Pause by allowing the Guardian to pause a specific
+chain/cluster as identified by a `SystemConfig` contract address. That is, the Guardian would call
+`SuperchainConfig.pause(someSystemConfigAddress)`. Other contracts would then check the status of
+the pause by calling `SuperchainConfig.paused(mySystemConfigAddress)` which would return true if
+either that specific system is paused or the entire Superchain is paused.
+
+The interop cluster can use a "fake" shared system config address in the call to `paused(..)` so
+that the entire cluster pauses at the same time.
+
+## Considerations
+
+### Pause Expiry Security Implications
+
+Pause Expiry has non-trivial security implications in that a bug in the proof system alongside a 
+malicious blocking minority of the Security Council could be catastrophic. L2Beat explciitly argues
+that this security model is better than the model in which a malicious Foundation and a blocking
+minority of the Security Model is catastrophic. Since this is fundamental to the updated Stage 1
+definition, we are required to accept this conclusion. We may want to take this as motivation to
+implement some other proof system as a backup to the primary proof system.
+
+### Permissioned Proof System
+
+A chain will be considered Stage 0 if it falls back to the Permissioned Proof System. We should
+accept this potential for a downgrade and generally rely on the pause mechanism as the primary
+incident response tool.
+
+We propose the following incident response protocol:
+
+- Always attempt to resolve an issue offchain if possible, before a game resolves incorrectly.
```

Comment:
what does that mean?

Reply:
Many times when you get an alert for an incident, the issue is with some offchain infrastructure and not a bug with a smart contract. If we can resolve the issue offchain before a game resolves incorrectly, do so. Somewhat self-evident to us internally but useful to make explicit about when we would and would not use a pause.

Reply:
```suggestion
- Always attempt to resolve an issue offchain if possible, before a game resolves incorrectly. 
  This would typically occur as a result of an issue in offchain infrastructure, rather than a bug in the contracts.
```